### DISCUSSION TITLE
Sonoff SONOFF ZIGBEE MINI - Cambiare modalità di attivazione (PULSE - EDGE - Follow)

### DISCUSSION THREAD

**Discussion by mrobotto-dev** [indomus/forum#1365] [Hardware & Firmware]
Ciao Ragazzi. Ho appena comprato un sonoff zigbee mini (modello ZBMINI); vorrei poterlo utilizzare con la modalità IMPULSO ma non ho modo di passare da ewelink non essendo possessore del Bridge della Sonoff.

Attualmente uso zigbee2mqtt tramite il dongle della Sonoff collegato ad home assistant e riesco ad utilizzarlo solo con gli interruttori.

Se comprassi un bridge della sonoff avrei la possibilità di switchare in modalità impulso per poi fare un repairing su zigbee2mqtt?

**Comment by Eddy16391** [comment]
Ciao, anche io ho lo stesso problema. Ci sto impazzendo. Nel frattempo hai trovato una soluzione?

**Reply by mrobotto-dev** [reply]
> 

Purtroppo no. Ho consultato tutti i siti, forum, video di ogni nazione ma nessuno pare abbia una soluzione. 

Sai come ho risolto? Utilizzando i nuovissimi shelly 1PM, praticamente sono fatti a posta per essere sostituiti ai relé. Non hai bisogno infatti di fare modifiche per l'utilizzo dei pulsanti, cosa invece necessaria per i sonoff.

I sonoff li uso quindi per solo per gli interruttori.

**Reply by Eddy16391** [reply]
Bene.. anzi male XD
Volevo usare gli ZBMINI perchè se presi dalla Cina stanno a nemmeno 9$ e rispetto agli Shelly il risparmio è notevole.
Però si potrebbe pensare anche di usarli insieme a un relè o a dei deviatori.
Staremo a vedere cosa mi propone l'elettricista.
Ti tengo aggiornato :-D

**Comment by sixdas** [comment]
Ciao alla fine siete riusciti a trovare una soluzione?
